docs/source/chapters/about.rst

```diff
@@ -47,22 +46,20 @@ Some of the pgRouting library core features are:
 * `Johnson's Algorithm <https://en.wikipedia.org/wiki/Johnson's_algorithm>`__
```

Comment:
Use documentation links
https://docs.pgrouting.org/latest/en/pgr_johnson.html

Reply:
Sounds perfect. Will change the links to pgRouting doc links.